README.md

```diff
@@ -455,6 +455,9 @@ Flattens array a single level deep.
   const flatten = [1, [2, [3, [4]], 5]].reduce( (a, b) => a.concat(b), [])
   // => [1, 2, [3, [4]], 5]
 
+  // Native(ES2019)
+  const flatten = [1, [2, [3, [4]], 5]].flat()
+  // => [1, 2, [3, [4]], 5]
```

Comment:
Add please support table for `Array​.prototype​.flat()`

Reply:
Okay, I append it